pkg/apis/validation/validation.go

```diff
@@ -51,6 +56,24 @@ func ValidateAttributes(attr map[string]string) error {
 	return nil
 }
 
+func keyUsages(ss string, errs []string) []string {
+	if len(ss) == 0 {
+		return errs
+	}
+
+	usages := strings.Split(ss, ",")
+
+	for _, usage := range usages {
```

Comment:
Maybe strip surrounding whitespace in case the user used `, ` as delimiter.

Reply:
:+1: